criticality_score/run.py

```diff
@@ -530,8 +530,10 @@ def get_repository(url):
         repo = None
         try:
             repo = get_github_auth_token().get_repo(repo_url)
+            # Validate whether repo is empty; if it's empty, calling totalCount throws a 409 exception
+            total_commits = repo.get_commits().totalCount
```

Comment:
I don't like adding another api call since it takes it out of quota. Is there an exception we can catch somewhere and bail out from furthur processing.